src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -1105,7 +1105,32 @@
     "useWpf": {
       "type": "computed",
       "datatype": "bool",
-      "value": "((!mauiEmbedding) && platforms == wpf)"
+      "value": "(platforms == wpf)"
+    },
+    "useNonMauiPlatforms": {
+      "type": "computed",
+      "datatype": "bool",
+      "value": "(platforms == wpf || platforms == wasm || platforms == gtk || platforms == linux-fb || platforms == macos)"
+    },
+    "xamlNamespaces": {
+      "type": "generated",
+      "generator": "switch",
+      "replaces": "$xamlNamespaces$",
+      "parameters": {
+        "evaluator": "C++",
+        "datatype": "string",
+        "cases": [
+          {
+            // For some reason this doesn't work: "condition": "(mauiEmbedding && useNonMauiPlatforms)"
+            "condition": "(mauiEmbedding && (platforms == wpf || platforms == wasm || platforms == gtk || platforms == linux-fb || platforms == macos))",
```

Comment:
@dansiegel can you see what I've missed here - the current condition works, but so should the condition in the comment but it doesn't seem to.